gmso/core/parametric_potential.py

```diff
@@ -141,29 +114,12 @@ def set_expression(self, expression=None, parameters=None, independent_variables
         If only a subset of the parameters are supplied, they are updated
         while the non-passed parameters default to the existing values
        """
-        self._potential_expression.set(
+        self.potential_expression_.set(
```

Comment:
Is the potential expression validated upon assignment? Or will that be part of another PR?

Reply:
It will be validated. However, it might be nice to check if its actually parametric i.e. it has parameters set correctly.